### PR TITLE
fix: add quoted msg to redux quotes when receiving quoted msg

### PR DESCRIPTION
**Pull request by Bilb**

### CODE REVIEW

File: ts/receiver/queuedJob.ts [hunk 1]
```diff
@@ -25,6 +29,12 @@ function contentTypeSupported(type: string): boolean {
   return Chrome.isImageTypeSupported(type) || Chrome.isVideoTypeSupported(type);
 }
 
+function isMessageModel(
+  msg: MessageModel | MessageModelPropsWithoutConvoProps
+): msg is MessageModel {
+  return (msg as MessageModel).get !== undefined;
```
Comment (yougotwill) [inline ts/receiver/queuedJob.ts:35]: 🧙

Comment (yougotwill) [reply ts/receiver/queuedJob.ts:35]: forgot this was my magic lmfao

**Review by yougotwill — APPROVED** [review]
LGTM! 🚢

File: ts/receiver/queuedJob.ts (hunk 1 above)
Comment (Bilb) [reply ts/receiver/queuedJob.ts:35]: well I cleaned it up by making it a typeguard, but yeah the `.get !== undefined` was from you :D

**Pull request merged**